server/authorizer/authorizer_test.go

```diff
@@ -67,6 +70,35 @@ func TestBasic(t *testing.T) {
             rights[_].path = input.path
         }
 
+		reason = "unauthorized method" {
+			not valid_method
+		}
+
+		reason = "unauthorized resource" {
+			not valid_path
+		}
+
+		reason = "unauthorized method & resource" {
+			not valid_method
+			not valid_path
+		}
+
+		allow = {
+			"allowed": false,
+			"reason": reason,
+		} {
+			not allow_inner
+			input.path[0] = "not_allowed_with_reason"
+		}
+
+        allow = { "allow": true } {
+			input.path[0] = "wrong_result_set"
+		}
+
+		allow = { "allowed": true } {
+			input.path[0] = "allowed_with_json_format"
+		}
```

Comment:
From what I can tell, these two could still be using the boolean variant we had before....? 🤔